editor/src/components/canvas/canvas-strategies/escape-hatch-strategy.tsx

```diff
@@ -416,3 +405,105 @@ function collectHighlightCommand(
   }, canvasState.selectedElements)
   return showOutlineHighlight('transient', [...siblingFrames, ...draggedFrames])
 }
+
+function findAbsoluteDescendants(
+  selectedElements: Array<ElementPath>,
+  metadata: ElementInstanceMetadataMap,
+): Array<ElementPath> {
+  const isSelectionInsideFocusedComponent = selectedElements.every((path) =>
+    EP.isInsideFocusedComponent(path),
+  )
+  return mapDropNulls((element) => {
+    const elementPath = element.elementPath
+    const isInsideFocusedComponent = EP.isInsideFocusedComponent(elementPath)
+    const allInsideComponentOrOutside =
+      isInsideFocusedComponent === isSelectionInsideFocusedComponent // if the selection and the children are both inside of the focused component
+
+    const sortedAncestors = EP.getOrderedPathsByDepth(selectedElements).filter((selection) =>
+      EP.isDescendantOf(elementPath, selection),
+    )
+    if (
+      MetadataUtils.isPositionAbsolute(element) &&
+      !EP.isRootElementOfInstance(elementPath) &&
+      sortedAncestors.length > 0 &&
+      allInsideComponentOrOutside
+    ) {
+      const closestAncestor = sortedAncestors[0]
```

Comment:
isn't this the same as EP.parentPath(elementPath) ?

Reply:
I was wondering the same thing here. Maybe a victim of some refactoring?